std_execution.bs

```diff
@@ -3022,3 +3039,152 @@ are all well-formed.
         does not call `std::terminate` if an error occurs after control is returned to the caller, the program is ill-formed with no diagnostic required.
 
     2. Otherwise, `execution::start_detached(execution::then(execution::schedule(sch), f))`.
+
+## Coroutine utilities <b>[execution.coro_utils]</b> ## {#spec-execution.coro_utils}
+
+### `execution::as_awaitable` <b>[execution.coro_utils.as_awaitable]</b> ### {#spec-execution.coro_utils.as_awaitable}
+
+1. `as_awaitable` is used to wrap a sender into an object that is awaitable within a particular coroutine. A sender `s` of type `S` can be made awaitable if and only if:
+
+    - `S` models the concept `typed_sender`, and
+
+    - It passes zero or one values of a unique type to `execution::set_value`, and
```

Comment:
Wording here is a little unclear. How do I pass zero values of a unique type? What is a unique type? What makes it unique?

Maybe change to something like:
- If the operation completes with `set_value()` then it calls at most one overload of `set_value()` with either
  zero or one value arguments.

Even then, is the fact that an operation completes with 1 type already covered by the `single-typed-sender` requirement below? Can we just say that `S` models `single-typed-sender` instead?